prealloc.go

```diff
@@ -304,16 +318,8 @@ func (v *returnsVisitor) Visit(node ast.Node) ast.Visitor {
 		return v
 
 	}
-	// if simple is true, then we actually have to check if we had returns inside of our loop. Otherwise, we can just report all messages.
-	if v.simple == true && !v.returnsInsideOfLoop {
-		for _, preallocMsg := range v.preallocMsgs {
-			log.Printf(preallocMsg)
-		}
-	} else if v.simple == false {
-		for _, preallocMsg := range v.preallocMsgs {
-			log.Printf(preallocMsg)
-		}
-	}
+	// if simple is true, then we actually have to check if we had returns
```

Comment:
Since we're doing processing on 116 now, I think this comment can go?

Reply:
Ah, yes it can.

Reply:
And so can that `default`